spk/borgbackup/src/requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Cross compiled packages dependencies
 #borgbackup==1.1.9
 #msgpack-python==0.5.6
-#ruamel.yaml==0.15.78
+ruamel.yaml==0.15.78
```

Comment:
It turns out that the cross compiled version shipped with python 3 is too recent here as compared to the implicit requirements (which basically enforce that it should be lower than 0.16.0